catalog/video/video-security/src/main/java/org/codice/alliance/video/security/claims/videographer/VideographerClaimsHandler.java

```diff
@@ -100,19 +100,21 @@ public ProcessedClaimCollection retrieveClaimValues(ClaimCollection claims,
             ClaimsParameters parameters) {
         ProcessedClaimCollection claimsColl = new ProcessedClaimCollection();
         Principal principal = parameters.getPrincipal();
-        for (Claim claim : claims) {
-            URI claimType = claim.getClaimType();
-            List<String> value = claimsMap.get(claimType);
-            if (value != null) {
-                ProcessedClaim c = new ProcessedClaim();
-                c.setClaimType(claimType);
-                c.setPrincipal(principal);
-                value.forEach(c::addValue);
-                claimsColl.add(c);
-            }
-        }
 
         if (principal instanceof VideographerPrincipal) {
+
+            for (Claim claim : claims) {
+                URI claimType = claim.getClaimType();
```

Comment:
I'm not sure we need a check for null. A null claimType just means the map.get on the next line will return null, and that value is null-checked.